setup.cfg

```
@@ -1,5 +1,54 @@
+[metadata]
+name = astroid
+version = attr: astroid.__pkginfo__.__version__
+description = An abstract syntax tree for Python with inference support.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/PyCQA/astroid
+author = Python Code Quality Authority
+author_email = code-quality@python.org
+license = LGPL-2.1
+license_file = LICENSE
```

```suggestion
license_files = LICENSE
```
I would suggest you remove the `pre-commit` hook. The config is unlikely to change any time soon anyway.

---

Both license_file and license_files are valid (and deprecated in favor of just giving the license in long_descripton) see https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html#metadata

---

Not entirely correct ;) https://setuptools.readthedocs.io/en/latest/history.html#v56-0-0
It's also mentioned here: https://setuptools.readthedocs.io/en/latest/references/keywords.html

`license_files` works better with `wheel` and also now supports glob pattern matching and defaults.

---

Ok, as we have a LICENSE we can remove this part entirely then ?

---

That should work. If not, I would have done something wrong. Although it's a bit more explicit with it being there.

---

Yeah I like the expliciteness too. I'm not used to that level of magic happening in setuptools 😄 Also, apparently the license_file deprecation is a little too fresh for setup-cfg-fmt to act on it, so I made a fork.